cmd/zfs/zfs_main.c

```diff
@@ -4333,16 +4336,6 @@ zfs_do_send(int argc, char **argv)
 		}
 	}
 
-	if (flags.dedup) {
-		(void) fprintf(stderr,
-		    gettext("WARNING: deduplicated send is "
-		    "deprecated, and will be removed in a\n"
-		    "future release. (In the future, the flag will be "
-		    "accepted, but a\n"
-		    "regular, non-deduplicated stream will be "
```

Comment:
Is this actually going into a separate release from the previous patch? If not, we might be rolling forwards with this a bit too quickly

Reply:
Yes, the previous commit which added this message is being backported to 0.8.3, but this commit won't be backported, it will be released with 2.0.